src/web/src/pages/NotFound.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div id="notfound">
+		<div class="notfound">
+			<div class="notfound-404">
+				<h1>Oops!</h1>
+				<h2>
+          <img src="../assets/images/sismanhigh.svg"/>
+          404 - The Page can't be found
+        </h2>
+			</div>
+			<a href="/">Go Back to YACS!</a>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+    name: "NotFoundPage",
+    components: {},
+    props: {},
+    data() {},
+    methods: {},
+}
+</script>
+
+<style>
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+h2 img{
+  position: absolute !important;
+  opacity: 0;
+  width: 750px;
+  height: 600px;
+  top: -500px;
+  left: -100px;
+  transition: opacity .5s;
+}
+
+h2:hover img {
+  opacity: 0.1;
```

Comment:
I wonder if he should be ever so slightly darker? Seems a little light even for an easter egg